master/buildbot/test/unit/test_www_service.py

```diff
@@ -274,3 +274,20 @@ def getHost(self):
                              algorithms=[service.SESSION_SECRET_ALGORITHM])
         self.assertEqual(decoded['user_info'], {'anonymous': True})
         self.assertIn('exp', decoded)
+
+    def test_absentServerHeader(self):
+
+        class FakeChannel:
```

Comment:
please factorize the FakeChannel

Reply:
Pardon? What is meant by that?

Reply:
The FakeChannel is copy paste from the previous test, so you could just put that class in the top level of that file, and share it between the two tests.